### PR TITLE
testing before - after component

### PR DESCRIPTION
**Pull request by gsalunke**
[Rendered Preview](https://fastncom-git-updated-before-after-fifthtry.vercel.app/blog/acme/)
@nandhinidevie Please, review the UI. @Arpita-Jaiswal requesting code review.

### CODE REVIEW

File: utils.ftd [hunk 1]
```diff
@@ -812,4 +812,67 @@ role: $inherited.types.copy-small
...
+
+-- component before-after:
+optional caption title: 
+boolean $show: false
+ftd.image-src before-image: 
+ftd.image-src after-image: 
+optional string before-caption:
+optional string after-caption:
+optional ftd.resizing width:
+optional ftd.resizing height:
```
Comment (Arpita-Jaiswal) [inline utils.ftd:825]:
```suggestion
optional caption title: 
ftd.image-src before-image: 
ftd.image-src after-image: 
optional string before-caption:
optional string after-caption:
optional ftd.resizing width:
optional ftd.resizing height:
boolean $show: false
```

File: utils.ftd [hunk 2]
```diff
@@ -812,4 +812,67 @@ role: $inherited.types.copy-small
...
+
+-- ftd.text: $before-after.title
+if: { before-after.title != NULL}
+role: $inherited.types.heading-small
+color: white
+if: { !before-after.show }
+anchor: parent
+top.percent: 50
+align-self: center
+background.solid: rgba(0,0, 0, 0.63)
```
Comment (Arpita-Jaiswal) [inline utils.ftd:844]:
```suggestion
background.solid: rgba(0, 0, 0, 0.63)
```

File: utils.ftd [hunk 3]
```diff
@@ -812,4 +812,67 @@ role: $inherited.types.copy-small
...
+
+-- ftd.column:
+if: { before-after.show }
+
+-- ds.image: $before-after.after-caption
+src: $before-after.after-image
+width: $before-after.width
+height: $before-after.height
+
+-- end: ftd.column
```
Comment (Arpita-Jaiswal) [inline utils.ftd:864]:
```suggestion
-- ds.image: $before-after.after-caption
src: $before-after.after-image
width: $before-after.width
height: $before-after.height
if: { before-after.show }
```

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

**Review by Arpita-Jaiswal — APPROVED** [review]

**Comment by nandhinidevie** [comment]
@gsalunke the before and after effect is not correct. The reader should be able to see both before and after images at all times. When the mouse is over the before image, the after image should be visible and when is mouse is not over the image, the before image should be visible again. 
Reference: On FifthTry's team page, each member's black and white image is the default, when the mouse is over the image, the colour photo is visible, when it is not it shows b&w image only.

**Comment by nandhinidevie** [comment]
The whatsapp screen suggestion for actual representation: 
Instead of Nandhini Devi, it should be ACME CEO, 
Instead of Ayush Soni, it should be a new phone number, 
the "Welcome to the Team.." message should not be from Meenu's, let's use Ajit's image and the name should be ACME VP- Marketing, 
Meenu's chat box should be different from Nandhini's, Ayush's and Ajit's chat box- Assume The Whatsapp group chat is from Meenu's phone. 
Another suggestion is to add the chat box inside an iPhone frame - see this image: https://www.pngkit.com/png/detail/776-7762990_whatsapp-features-whatsapp-in-ios-screen.png

**Pull request merged**